lib/language.js

```diff
@@ -214,7 +225,8 @@ function init() {
       ,dk: 'Onsdag'
       ,fi: 'Keskiviikko'
       ,nb: 'Onsdag'
-      ,he: 'רביעי'
+      ,he: 'רביעי
+      ,nl: 'Woensdag''
       }
     ,'Thursday' : {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected an assignment or function call and instead saw an expression.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/2496909485/issues/source?bid=2452374&fileBranchId=3269876#l231)